gapis/resolve/find.go

```diff
@@ -76,12 +76,15 @@ func Find(ctx context.Context, req *service.FindRequest, h service.FindHandler)
 			return err
 		}
 
-		nodePred := func(item api.CmdIDGroupOrID) bool {
+		nodePred := func(item api.CmdIDGroupOrIDOrRoot) bool {
 			switch item := item.(type) {
 			case api.CmdIDGroup:
 				return pred(item.Name)
-			case api.CmdID:
-				return pred(fmt.Sprint(c.Commands[item]))
+			case api.SubCmdIdx:
+				if len(item) > 1 {
+					panic("todo: Support subcommands")
```

Comment:
You happy with the fact you can kill gapis using the search bar in a Vulkan trace? :smile: